vendor.mod

```diff
@@ -97,32 +97,33 @@ require (
 	github.com/vbatts/tar-split v0.12.1
 	github.com/vishvananda/netlink v1.3.1
 	github.com/vishvananda/netns v0.0.5
-	go.etcd.io/bbolt v1.3.11
-	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.56.0
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.56.0
+	go.etcd.io/bbolt v1.4.0
```

Comment:
Looks like this update was pending in https://github.com/moby/moby/pull/50031 - we could merge that one first if we want /cc @vvoland